region/norfair/crocomire.json

```diff
@@ -1624,7 +1624,8 @@
                   "requires": [
                     "canUseFrozenEnemies",
                     "canManipulateMellas",
-                    "HiJump"
+                    "HiJump",
+                    "canConsecutiveWalljump"
```

Comment:
This only takes 1 walljump, might even be able to do it with none.

Reply:
I think it should take 2 wall jumps to get onto the Mella and then one more to get from the Mella to the item.
https://youtu.be/a02RaagZqjo

Reply:
You linked the no HiJump version.
With hijump its jump up to the bug then jump into walljump.

Reply:
Well, that's embarrassing.